### PR TITLE
Update URL's in Salt manual

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: plugins/foreman_salt/3.0/index.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ If using the Foreman installer, install the core and smart proxy plugins with:
 foreman-installer --enable-foreman-plugin-salt --enable-foreman-proxy-plugin-salt
 ```
 
-For manual installation, see [How to Install a Plugin](http://www.theforeman.org/manuals/latest/index.html#6.1InstallaPlugin) for more info.
+For manual installation, see the [plugin manual](http://www.theforeman.org/plugins/#2.Installation) for more info.
```
Comment (domcleal) [inline plugins/foreman_salt/3.0/index.md]: Could you remove the domain as well please?  `{{site.baseurl}}plugins/..` ought to work.

**Comment by stbenjam** [comment]
Fixed

File: plugins/foreman_salt/3.0/index.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ If using the Foreman installer, install the core and smart proxy plugins with:
 foreman-installer --enable-foreman-plugin-salt --enable-foreman-proxy-plugin-salt
 ```
 
-For manual installation, see [How to Install a Plugin](http://www.theforeman.org/manuals/latest/index.html#6.1InstallaPlugin) for more info.
+For manual installation, see the [plugin manual]({{ site.baseurl }}/plugins/#2.Installation) for more info.
```
Comment (domcleal) [inline plugins/foreman_salt/3.0/index.md]: The `/` between the baseurl and plugins needs to be removed, it's just linking to `//` with it.

Comment (stbenjam) [inline plugins/foreman_salt/3.0/index.md]: Fixed, sorry, should've checked.

**Comment by domcleal** [comment]
Thanks!

**Pull request merged**